incubator/moodle/values.yaml

```diff
@@ -0,0 +1,80 @@
+## Bitnami moodle image version
+## ref: https://hub.docker.com/r/bitnami/moodle/tags/
+##
+image: bitnami/moodle
+
+## Specify a imagePullPolicy
+## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+##
+# imagePullPolicy:
+
+## User of the application
+## ref: https://github.com/bitnami/bitnami-docker-moodle#environment-variables
+##
+moodleUser: user
+
+## Application password
+## Defaults to a random 10-character alphanumeric string if not set
+## ref: https://github.com/bitnami/bitnami-docker-moodle#environment-variables
+moodlePassword: password
```

Comment:
the `moodlePassword` defaults to a random string, the param here should be commented